integration/kubernetes/nginx.bats

```diff
@@ -32,7 +32,7 @@ setup() {
 	busybox_pod="test-nginx"
 	kubectl run $busybox_pod --restart=Never --image="$busybox_image" \
 		-- wget --timeout=5 "$deployment"
-	cmd="kubectl get pods -a | grep $busybox_pod | grep Completed"
+	cmd="kubectl get pods | grep $busybox_pod | grep Completed"
```

Comment:
hey @GabyCT - did you also sneak a bugfix in here - as I just found `kubectl get pods -a` is not a valid `kubectl` cmd :-)
Probably should have dropped that on a separate commit in this PR.
Was this an effect of k8s updating, or has it always been broken? If broken, how did it ever get through the CI/bats ?
thx!

Reply:
@grahamwhaley it was an effect of k8s updating to a newer version